docs/manual-instrumentation.md

```diff
@@ -28,12 +28,12 @@ use the `@WithSpan` annotation, also include the `opentelemetry-extension-annota
     <dependency>
       <groupId>io.opentelemetry</groupId>
       <artifactId>opentelemetry-api</artifactId>
-      <version>1.0.0</version>
+      <version>1.7.0</version>
```

Comment:
until we've updated to the 1.7.0 javaagent, we should keep this at 1.6.0, I think.